pkg/ormb/ormb.go

```diff
@@ -80,6 +81,11 @@ func (o ORMB) Export(refStr, dst string) error {
 		return err
 	}
 
+	err = os.MkdirAll(path, os.ModePerm)
+	if err != nil {
+		return err
```

Comment:
If file/dir exists already, can log only? not `return err`?

Reply:
it will do nothing if file/dir exists already, not return err.

Reply:
Can export normally even dir is exists?

Reply:
Yes, see the test result below

Reply:
SGTM, thanks for you contributions